src/common/dao/registry.go

```diff
@@ -0,0 +1,107 @@
+package dao
```

Comment:
As we discussed, the dao methods should be put under directory `replication/ng`, can we move them in this PR?

Reply:
Moved.